shell/detail/helm.cattle.io.projecthelmchart.vue

```diff
@@ -211,7 +198,7 @@ export default {
           </div>
           <a
             :class="{disabled: !grafanaServiceEndpointEnabled}"
-            :href="value.status.dashboardValues.grafanaURL"
+            :href="grafanaURL"
```

Comment:
In both flavours mentioned in review description... this lead to what looks like a default home page.

![image](https://github.com/rancher/dashboard/assets/18697775/602df085-5f55-449f-ab96-3211375a3690)

The dashboard links do end with a `proxy/` but is redirected by something grafana side to a url ending with `proxy/?orgId=1`. Is that waiting a BE fix?